src/wallet/wallet.cpp

```diff
@@ -576,7 +576,8 @@ std::optional<libzcash::ZcashdUnifiedSpendingKey>
             throw std::runtime_error("CWallet::GenerateUnifiedSpendingKeyForAccount(): Failed to add Sapling change address to the wallet.");
         };
 
-        // TODO ORCHARD: Add Orchard component to the wallet
+        // Add Orchard spending key to the wallet
+        orchardWallet.AddSpendingKey(usk.value().GetOrchardKey());
```

Comment:
To clarify, this change means that `z_getnewaccount` will start storing the Orchard component in memory, but this PR is not exposing this fact in `z_getnewaccount` or allowing Orchard-containing UAs to be derived. Those changes will be made in a subsequent PR.